vendors/circom/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+load("@kroma_network_tachyon//bazel:tachyon_cc.bzl", "tachyon_cc_binary")
+
+tachyon_cc_binary(
+    name = "prover_main",
+    srcs = ["prover_main.cc"],
+    deps = [
+        "//circomlib/circuit:quadratic_arithmetic_program",
+        "//circomlib/json",
+        "//circomlib/json:groth16_proof",
+        "//circomlib/json:prime_field",
```

Comment:
Can you delete these two since line 8 takes care of them?

Reply:
You mean `"//circomlib/json"`?

Reply:
hmm I'm pretty curious on how bazel works, but I thought that `"//circomlib/json"` introduces all the possible targets in the circomlib/json folder, while `"//circomlib/json:json"` would import the target you mentioned.

Reply:
You can think like "//circomlib/json" equals to "//circomlib/json:json".